workers/unity/Packages/com.improbable.gdk.gameobjectcreation/GameObjectCreationSystemHelper.cs

```
@@ -8,7 +8,7 @@ public static class GameObjectCreationSystemHelper
     {
         public static void EnableStandardGameObjectCreation(World world, GameObject workerGameObject = null)
         {
-            var workerSystem = world.GetOrCreateManager<WorkerSystem>();
+            var workerSystem = world.GetExistingManager<WorkerSystem>();
```

should we throw an exception if this returns null?

---

this should already do that :)

---

`GetExistingManager` won't throw an exception, it will return null